### ISSUE TITLE
csmith: Statement.cpp:936: void Statement::post_creation_analysis(std::vector<const Fact*>&, const Effect&, CGContext&) const: Assertion `0' failed.

### ISSUE THREAD

**Issue by yansendao** [csmith-project/csmith#98]
```
/*
 * This is a RANDOMLY GENERATED PROGRAM.
 *
 * Generator: csmith 2.4.0
 * Git version: 5039909
 * Options:   (none)
 * Seed:      1599421700885742
 */

#include "csmith.h"


static long __undefined;




$ csmith -v
csmith 2.4.0
Git version: 5039909
```

**Comment by jxyang** [comment]
Seems to be a duplicate of 104. Thanks for reporting.

**Comment by jxyang** [comment]
Fixed with PR #107. Thanks for reporting.
